src/index.js

```diff
@@ -1,6 +1,21 @@
 const fibonacci = (n) => {
+    let fibArray = [];
+    for (let i = 1; i < n + 1; i++) {
+      fibArray.push(recursive(i));
+    }
+  
+    return fibArray;
+  };
+  
+  let recursive = function(i) {
+    if (i < 2) {
+      return i;
+    }
+    else {
+      return recursive(i - 1) + recursive(i - 2);
+    }
+  };
```

Comment:
Este algoritmo es un poco ineficiente, ya que en cada paso del for de arriba vas calculando toda la serie anterior, que ya tienes calculada. Intentar con recursividad y programación dinámica. Adapta la idea que David hizo en python a javascript de la case https://platzi.com/clases/1835-programacion-estocastica/26430-optimizacion-de-fibonacci/

Reply:
De acuerdo el fin de semana lo trabajo. Gracias por los comentarios.